src/main/scala/com/databricks/labs/overwatch/pipeline/BronzeTransforms.scala

```diff
@@ -897,17 +898,17 @@ trait BronzeTransforms extends SparkSessionWrapper {
 
 
   private[overwatch] def getAllEventLogPrefix(inputDataframe: DataFrame, apiEnv: ApiEnv): DataFrame = {
+    try{
     val mountMap = getMountPointMapping(apiEnv) //Getting the mount info from api and cleaning the data
+      .withColumn("mount_point", when('mount_point.endsWith("/"), 'mount_point.substr(lit(0), length('mount_point) - 1)).otherwise('mount_point))
```

Comment:
wasn't this implemented in #699 ? Curious as to why it's in this PR too. No problem, just want to confirm the logic is exactly the same. 

Added #768 for this

Reply:
its a different issue when get the source to mount mapping using dbutils i see the data which is present in csv contains the "/" at the end hence some of the joins were failing. https://github.com/databrickslabs/overwatch/pull/699 contains the "/" issue only in apiURL.

Reply:
ahh i see, thanks for clarifying